src/new.md

```diff
@@ -3,6 +3,32 @@ layout: page
 permalink: /resources/new/
 title: What's New
 ---
+<style>
```

Comment:
- styles should be in /assets/scss/styles.css
- there's existing styles for buttons (see bootstrap css), just need to add the existing class